README.md

```diff
@@ -40,28 +42,28 @@ Another aspect is to keep a repository's meaning clear; for example. don't combi
 How best to structure a repository is very dependent upon what is being developed.
 A good way to structure a repo is to place everything needed for a solution within that repository.
 For example, place a Dockerfile used to build an image in the repo and not in a separate repo.
-If there are multiple Dockerfiles for a single repo, you may consider putting them in a folder named "dockerfiles", but then again maybe not.
+If there are multiple Dockerfiles for a single repo, you may consider putting them in a folder named "dockerfiles", but this is not required.
 
-Keeping repos as single purpose as possible is also encouraged.
-This does not prevent major complexity within a repo.
+Keeping repositories as single purpose as possible is also encouraged.
+This does not prevent major complexity within a repository.
 
 In the end, use your best judgement, don't be afraid to reorganize, and ask if you have any questions!
 
 # Issues
 
 We are run by the issues that are entered into our system.
```

Comment:
I know this isn't included in the PR, but this sentence is unclear to me. If I'm understanding correctly, would something like 
```suggestion
Our work is directed by the issues submitted.
```
make more sense?

Reply:
Definitely more sense has been made! Thanks!